replica/table.cc

```diff
@@ -1651,7 +1651,9 @@ future<> table::flush(std::optional<db::replay_position> pos) {
 }
 
 bool table::can_flush() const {
-    return _compaction_group->can_flush();
+    return std::ranges::all_of(compaction_groups(), [] (const compaction_group_ptr& cg) {
+        return cg->can_flush();
```

Comment:
Similar, can use member pointer.

Reply:
I don't think it was a bug, but rather partial support which resulted in compiler errors for unsupported features.

https://libcxx.llvm.org/Status/Cxx20.html